src/meshes/mesh_io.jl

```diff
@@ -119,6 +120,29 @@ function save_mesh_file(mesh::StructuredMesh, output_directory; system = "")
     return filename
 end
 
+# Does not save the mesh itself to an HDF5 file. Instead saves important attributes
+# of the mesh, like its size and the type of boundary mapping function.
+# Then, within Trixi2Vtk, the StructuredMesh and its node coordinates are reconstructured from
+# these attributes for plotting purposes.
+function save_mesh_file(mesh::StructuredMeshView, output_directory; system = "",
```

Comment:
Since this is still very experimental, would you mind moving this new method to the `src/meshes/structured_mesh_view.jl` file such that we have all truly new functions/types in a single files (similar like we did with the semi coupled)?

Reply:
Done.